### DISCUSSION TITLE
Help translate Jamulus!

### DISCUSSION THREAD

**Discussion by gilgongo** [jamulussoftware/jamulus#2057] [Announcements]
**The Jamulus project has always aspired to be international, and people across several continents currently help to maintain it. We're proud to have several language translations of both the Jamulus application itself and the website.** 

If you're familiar with application translation and can help out, let us know! We've recently updated the website (using po4a) to make it a similar process to that of the application too.

Our documentation for [application translations is here](https://github.com/jamulussoftware/jamulus/blob/master/TRANSLATING.md)  

Details on [how to translate the website are here](https://github.com/jamulussoftware/jamuluswebsite/tree/release/_translator-files). 

_We're more than happy to provide help and support, even if you can't commit to translating everything into your language._

**Comment by ann0see** [comment]
We’re currently missing a few languages. Have a look at the issues tab.

E.g. German is missing as far as I know. There are many Germans using Jamulus, so it would be great to have at least the app translated!

**Reply by sepplfredo** [reply]
I am from Southern Germany and I am using Jamulus. I would like to contribute some translation work for DE. I am currently doing some preparations for this.

**Reply by ann0see** [reply]
Great! Thank you very much. If you need help, just open an issue on the website or the app.
